custom/ilsgateway/tanzania/reports/facility_details.py

```diff
@@ -136,27 +137,68 @@ def rows(self):
             ]
 
 
+def _fmt_timestamp(timestamp):
+    return dict(
+        sort_key=timestamp,
+        html=timestamp.strftime("%Y-%m-%d %H:%M:%S"),
+    )
+
+
+def _fmt(val):
+    if val is None:
+        val = '-'
+    return dict(sort_key=val, html=val)
+
+
 class RecentMessages(ILSData):
-    slug = 'ils_notes'
+    slug = 'recent_messages'
     title = 'Recent messages'
     show_chart = False
     show_table = True
+    default_rows = 5
 
     @property
     def headers(self):
         return DataTablesHeader(*[
-            DataTablesColumn()
+            DataTablesColumn('Date'),
+            DataTablesColumn('Direction'),
+            DataTablesColumn('Text')
         ])
 
+    @property
+    def rows(self):
+        data = SMSLog.by_domain_date(self.config['domain'])
```

Comment:
@lwyszomi @kkrampa this should use the `corehq.apps.sms.models.SMS` SQL model and do the location filtering in SQL